docs/faq.md

```diff
@@ -157,3 +157,9 @@ You can read more about this transition [here](/posts/copr-srpms).
 ## A command failed in packit-service: how do I reproduce it locally?
 
 Please read [this document on how to reproduce locally](reproduce-locally).
+
+## Unlocking Efficiency: The Advantages of Using Packit for Seamless Service Integration
+
+- Packit uses various services Testing Farm, Copr, Koji, Bodhi…
+- These services can be used without Packit, but Packit intergrates their continuity (the build in Copr is created automatically when a new version is released).
```

Comment:
```suggestion
- These services can be used without Packit, but Packit integrates them (the build in Copr is created automatically when a new version is released).
```
My english is bad, but the sentence above sounds to me better in the way I propose, but feel free to ignore my suggestion (just apply the typo fix :-))

Reply:
Sounds better to me :eyes:

Reply:
I would also maybe explicitly emphasise that in some workflows (mainly Fedora maintenance automation), we are chaining the tasks run on these services so people do not need to be busy waiting.

Reply:
+1, I would in general extend this a little bit more, e.g. with how we take care of passing the built NVRs of RPM builds to Testing Farm